Source/WebCore/css/CSSPseudoSelectors.json

```
@@ -19,7 +19,12 @@
             ],
             "user-agent-part": [
                 "For pseudo-elements only. Whether the pseudo-element represents an element in an user agent shadow tree.",
-                "They internally map to `PseudoElement::UserAgentPart` or `PseudoElement::UserAgentPartLegacyAlias` for aliases."
+                "They internally map to `PseudoElement::UserAgentPart` or `PseudoElement::UserAgentPartLegacyAlias` for aliases.",
+                "Also causes an entry in UserAgentPartIds to be created."
```

```suggestion
                "Also creates an entry in UserAgentPartIds."
```
nit: "causes" reads as if it's a negative side-effect.